src/components/RequiredAsterisk.vue

```diff
@@ -24,6 +24,8 @@ export default {
 @import "@wmde/wikit-tokens/variables";
 
 .wbl-snl-required-asterisk {
+	font-size: $font-size-xxlarge;
```

Comment:
I think this should probably also add
```css
line-height: 0;
```
otherwise it significantly bumps the line height of the whole label, which affects the layout.

Reply:
Before/after:
![image](https://github.com/wmde/new-lexeme-special-page/assets/2346599/8d1e5f47-1ae1-4c3f-8535-b866e99fa928)

Reply:
That looks much better indeed. I also like that now the asterisk has a bit of a "superscript" feel to it.